docs/src/man/getting_started.md

```diff
@@ -877,6 +877,58 @@ package provides interfaces similar to LINQ and [dplyr](https://dplyr.tidyverse.
 
 See the [Data manipulation frameworks](@ref) section for more information.
 
+#### Selecting rows with `filter` and `subset`
```

Comment:
I'd put this section above the one presenting `select` and `transform`, which is more complex and goes beyond taking subsets. (Transformations probably deserve a separate section so that we only present columns selection, but that's another story.)

That way you don't need to repeat the example: you can just continue with the ones given at the end of the Indexing section.

Reply:
We cannot, as `subset` has the same syntax as `select` so we need to introduce it after `filter`. We could split the discussion of `filter` and `subset` though.

Reply:
Ah indeed. Let's keep it that way then. But "Taking a Subset" isn't actually a good name for the parent section as it also shows complex transformations. Maybe we should just split each sub-section into its own section, i.e. "Indexing", "Selecting and transforming columns" and "Selecting rows".

Reply:
Agreed.